src/utils/line.ts

```diff
@@ -290,39 +308,85 @@ export const formatSecuredLineData = (
       positionIds: Object.keys(positions),
       positions,
     },
+    creditEvents,
     escrow,
     spigot: { type: COLLATERAL_TYPE_REVENUE, line, tokenRevenue },
   };
 };
 
+// TODO: Modify/adjust this to replace mergeCollateralEvents (formatCollateralEvents)
+// given that there are 3 duplicative uses of collateral events in this file!!
+export const formatEscrowToCollateralEvents = (escrow: AggregatedEscrow | undefined): CollateralEvent[] => {
+  const collateralEvents: CollateralEvent[] = escrow!
+    ? _.map(escrow.deposits, function (deposit) {
+        return {
+          type: escrow.type,
+          id: deposit.token.address,
+          // TODO: timestamp should probably be removed from the type given this aggregates
+          // collateral by type and token address
+          timestamp: 0,
+          amount: Number(deposit.amount),
+          // TODO: replace with correct USD value when we have it
+          value: 0,
+        } as CollateralEvent;
+      })
+    : [];
+  return collateralEvents;
+};
+
+export const formatLineWithEvents = (
```

Comment:
Cant you just do `formatGetLinesEvents(lineEvents)` and merge return value directly into SecuredLineWithEvents no need for a whole function?